### DISCUSSION TITLE
Unknown changes Improvements in 0.96-beta4

### DISCUSSION THREAD

**Discussion by ineuw** [autokey/autokey#581] [Beta feedback]
How can we find out which library is no longer needed to be imported like "time", "mouse", etc..? Or, any other changes and improvements?  Though, I am aware that documenting changes can take longer than programming the features.  Is there a new beta version number planned?

**Comment by josephj11** [comment]
Skipping `import time` works in 0.95.10, so it's not a change to the currenti release. I don't know when/if that change happened.

If you use a module that isn't available, you get a traceback that tells you it couldn't find it, so it's easy to see that an import statement is needed.

As for beta version numbers, whenever there are "enough" fixes to justify a new beta release, Bluedrink9 will just make one and release it with a post on Gitter. That will continue until the beta is ready to be the next production release at which point, the whole cycle starts over - except that then, there won't be a new beta release at all - just the development branch until it looks like enough progress has been made to warrant a new release with its own beta.
